Jenkinsfile

```diff
@@ -0,0 +1,79 @@
+readTrusted 'Dockerfile_jenkins_ubuntu32'
+
+pipeline {
+  agent none
+
+  stages {
+    stage('64bit') {
+      agent {
+        docker {
+          image 'golang:1.9.2'
+        }
+      }
+
+      steps {
+        sh """
+        mkdir -p /go/src/github.com/influxdata
+        cp -a $WORKSPACE /go/src/github.com/influxdata/influxdb
+
+        cd /go/src/github.com/influxdata/influxdb
+        go get github.com/sparrc/gdm
+        gdm restore
+        """
+
+        sh """
```

Comment:
Why are these separate `sh` blocks instead of one contiguous `sh` block?

Reply:
I think there's going to be a feature in the future to name the blocks. I wanted it so these would be shown as separate steps in the UI.